src/spector.ts

```diff
@@ -303,8 +305,8 @@ export class Spector {
             this.capturingContext = contextSpy;
             this.capturingContext.setMarker(this.marker);
 
-            // Limit command count to 5000 record.
-            commandCount = Math.min(commandCount, 5000);
+            // Limit the shown command count
```

Comment:
nit: Other code in this file ends the comment with `.` (as did the old code). Minor inconsistency only.